app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+    <% if @minimum_password_length %>
+      <br />
+      <em><%= @minimum_password_length %> characters minimum</em>
+    <% end %>
```

Comment:
- [OPTIONAL] Good job implementing the authentication however your login and signup pages should look like the provided wireframes, kindly make sure to implement this so your project is completed. 💯 👍🏻 

|   |   |
|-|-|
|![image](https://user-images.githubusercontent.com/79058364/217609913-0e1b3629-aefc-4428-ac98-ca146274ad83.png)|![image](https://user-images.githubusercontent.com/25227022/216383255-33d4fd58-29e6-417b-9b68-f97b4f6597dd.png)|